|-> error(message) {
    print("Error!")
    print(message)
}